distribution/argocd-applications/aws-load-balancer-controller.yaml

```diff
@@ -15,6 +15,10 @@ spec:
           value: <<__role_arn.loadbalancer_controller__>>
         - name: clusterName
           value: <<__cluster_name__>>
+        - name: image.repository
+          value: amazon/aws-alb-ingress-controller
+        - name: image.tag
+          value: v2.2.0    
```

Comment:
While it probably is best practice to define an image tag, I'm pretty sure this will somewhat break Renovate updates because the image tag will stay static while the helm chart is updated.

Reply:
Good point. I guess this wasn't even necessary either, would have been enough to just set the image repo as the chart defaults to 2.2.0 in any case